### PR TITLE
Close InputStream after reading it

### PR DESCRIPTION
**Pull request by gastaldi**
@dmlloyd Can you review it?

### CODE REVIEW

File: src/main/java/org/jboss/modules/FileResourceLoader.java [hunk 1]
```diff
@@ -94,11 +94,18 @@ public URL run() {
     }
 
     private static Manifest readManifestFile(final File manifestFile) {
+        Manifest manifest = null;
+        FileInputStream fis = null;
         try {
-            return manifestFile.exists() ? new Manifest(new FileInputStream(manifestFile)) : null;
```
Comment (dmlloyd) [inline src/main/java/org/jboss/modules/FileResourceLoader.java]: Instead maybe just use try-with-resources?

Comment (gastaldi) [inline src/main/java/org/jboss/modules/FileResourceLoader.java]: Is it being compiled with JDK7 ? I missed that.

Comment (gastaldi) [inline src/main/java/org/jboss/modules/FileResourceLoader.java]: Try-with-resources still asks to catch IOException. I think this is fine

**Comment by gastaldi** [comment]
@dmlloyd PR updated, please review. I wasn't sure about the code formatter used, so I used the Java Conventions (Built-in) in Eclipse

**Pull request merged**